<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_algorithm</class>
 <widget class="QDialog" name="dialog_algorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Disc Cover Algorithm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_algorithm">
       <property name="text">
        <string>Algorithm:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_box_algorithm">
       <item>
        <property name="text">
         <string>Isolated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Naive</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Greedy Naive</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clique Grredy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IP Max Clique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IP All Cliques</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hochbaum &amp; Mass Approximation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biniz et al. Approximation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Max Cliques AMPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Export AMPL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_use_time_limit">
       <property name="text">
        <string>Use Time Limit:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="check_box_use_time_limit">
       <property name="text">
        <string>Yes / No</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_time_limit">
       <property name="text">
        <string>Time Limit (seconds):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_time_limit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>dialog_algorithm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>dialog_algorithm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
